### PR TITLE
[Merged by Bors] - refactor(TensorProduct): golf the definition of Neg

### PR DESCRIPTION
**Pull request by ChrisHughes24**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/LinearAlgebra/TensorProduct.lean [hunk 1]
```diff
@@ -1234,33 +1234,16 @@ open LinearMap
 variable (R)
 
 /-- Auxiliary function to defining negation multiplication on tensor product. -/
-def Neg.aux : FreeAddMonoid (M × N) →+ M ⊗[R] N :=
-  FreeAddMonoid.lift fun p : M × N => (-p.1) ⊗ₜ p.2
-#align tensor_product.neg.aux TensorProduct.Neg.aux
+def Neg.aux : M ⊗[R] N →ₗ[R] M ⊗[R] N :=
+  lift <| (mk R M N).comp (-LinearMap.id)
```
Comment (eric-wieser) [inline Mathlib/LinearAlgebra/TensorProduct.lean:1238]: I'd be tempted to inline this below

**Review by eric-wieser — APPROVED** [review]
bors d+

Thanks!

File: Mathlib/LinearAlgebra/TensorProduct.lean (hunk 1 above)
Comment (ChrisHughes24) [reply Mathlib/LinearAlgebra/TensorProduct.lean:1238]: I think that makes the proof slightly longer.

**Pull request closed without merging**